packages/composites/proof-of-reserves/src/utils/addressValidator.ts

```diff
@@ -0,0 +1,52 @@
+import { utils } from 'ethers'
+import { Logger } from '@chainlink/ea-bootstrap'
+
+type AddressObject = { address: string }
+type AddressArray = AddressObject[]
+
+export const validateAddresses = (indexer: string, addresses: AddressArray): AddressArray => {
+  switch (indexer) {
+    case 'eth_balance':
```

Comment:
Assuming a `network` param is passed with each address, can we switch on that primarily? And if it's not present, fall back to the indexer?

I'm afraid of us forgetting to put in the right indexer here. Another problem is that we have indexers that work for multiple blockchains (bitcoin + dogecoin).